test/infamy/iface.py

```diff
@@ -0,0 +1,54 @@
+def extract_iface_param(row, key):
```

Comment:
Minor, all class methods should have a docstring right after the `def method` line, for example:

    """Extract foo from bar"""

Also, member names should be descriptive, so it's easier to understand from the method name what they do, which in turn makes it easier to write the docstring.  See more in PEP-8 https://peps.python.org/pep-0008/#documentation-strings